components/reusable-components/reusable-table/TableRowComponent.tsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-unused-vars */
+
+import React from 'react';
+import style from './tableRowComponent.module.scss';
+import ActionDropdown from './ActionDropdown';
+import { UserData } from './ReusableTable';
+
+interface DropdownItem {
+  id: string;
+  label: string;
+}
+
+interface TableRowComponentProps<T extends UserData> {
```

Comment:
we shouldn't extend UserData, this way the Table Row is not reusable

Reply:
We should also consider the following scenario:
what if we don't want to show every entry from the data in the table.
Example we are displaying the blog posts. Each blog post contains: id, title, description, slug, imageUrl, author, createdAt, updatedAt
But in the table we want to just display: title, description, imageUrl, author.

Reply:
Need some time to implement this scenario, I hope will be very soon...

Reply:
you would need another key in the TableRowProps interface that would be an []
in the ReusableTable you will pass the headers in that prop